scripts/ci-checks.sh

```diff
@@ -89,7 +89,7 @@ fi
 
 source scripts/env/bin/activate
 pip install -U pip
-pip install -U wheel black pylint mypy 1>/dev/null
+pip install -U wheel black==22.12.0 pylint==2.15.10 mypy 1>/dev/null
```

Comment:
Both `black` and `pylint` have recently had breaking updates, and we don't want to apply those changes to 2.x. This pins them to compatible versions.